README.md

```diff
@@ -46,18 +46,30 @@ python3 install.py <flavor> <accent>
   [catppuccin-gtk] [INFO] - Theme installation complete! 
 ```
 
+### Flatpak
+In order to give flatpaks access to the theme directory, run this command:
+```bash
+sudo flatpak override --filesystem=$HOME/.local/share/themes
+```
+
+Then, run this command to set the theme:
+```bash
+export THEME_DIR="~/.local/share/themes/catppuccin-<flavor>-<accent>-standard+default"
+sudo flatpak override --env=GTK_THEME=$THEME_DIR
+```
+
 ### Arch Linux
-  With your favourite AUR helper, you can install your theme of choice:
+With your favourite AUR helper, you can install the 4 flavors (Latte, Frappe, Macchiato, Mocha):
 
-  ```bash
-  yay -S catppuccin-gtk-theme-<flavor>
-  paru -S catppuccin-gtk-theme-<flavor>
-  ```
+```bash
+yay -S catppuccin-gtk-theme-<flavor>
+paru -S catppuccin-gtk-theme-<flavor>
+```
 
 ### Nix
-  We have created a Nix module ([catppuccin/nix](https://github.com/catppuccin/nix)) for theming apps under Nix, and recommend that you use it.
-  You can set up our Nix module for GTK with the following config:
-  ```nix
+We have created a Nix module ([catppuccin/nix](https://github.com/catppuccin/nix)) for theming apps under Nix, and recommend that you use it.
+You can set up our Nix module for GTK with the following config:
+```nix
   {inputs, ...}: {
```

Comment:
can you remove 1 level of indenting on the nix code.

Reply:
sure

Reply:
The rest is still indented